tailwind.config.js

```diff
@@ -122,6 +122,9 @@ module.exports = {
       },
       width: {
         'form': '45.625rem'
+      },
+      margin: {
+        '84': '24rem'
```

Comment:
this is too much specific of a rewrite on our rules. Usually there is another way to get the usual "margin" behaviour instead of using gigantic spacing. Can you explain when you use it and what is the specific thing you wanted to implement using this?